src/Compilers/CSharp/Test/Semantic/Semantics/AnonymousFunctionTests.cs

```diff
@@ -0,0 +1,802 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Linq;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.CSharp.Test.Utilities;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Roslyn.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests
+{
+    public class AnonymousFunctionTestBase : CSharpTestBase
+    {
+        internal static readonly CSharpParseOptions DefaultParseOptions = TestOptions.Regular;
+
+        internal static void VerifyDiagnostics(string source, params DiagnosticDescription[] expected)
+        {
+            var comp = CreateCompilationWithMscorlib45AndCSharp(source, options: TestOptions.ReleaseDll, parseOptions: DefaultParseOptions);
+            comp.VerifyDiagnostics(expected);
+        }
+    }
+
+    [WorkItem(275, "https://github.com/dotnet/csharplang/issues/275")]
+    [CompilerTrait(CompilerFeature.AnonymousFunctions)]
+    public class AnonymousFunctionTests : AnonymousFunctionTestBase
+    {
+        [Fact]
+        public void StaticLambdaCanCaptureStaticField()
+        {
+            var source = @"
+using System;
+
+public class C
+{
+    public static int a;
+
+    public void F()
+    {
+        Func<int> f = static () => a;
+    }
+}";
+            CreateCompilation(source).VerifyDiagnostics(
```

Comment:
>source [](start = 30, length = 6)

Consider using `parseOptions: TestOptions.RegularPreview` for each of these tests that are not directly testing `ERR_FeatureInPreview`.

Reply:
will do!